pkgs/tools/security/enpass/update_script.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import print_function
```

Comment:
well, it has to load stuff from a debian repo - there is no easy install able tarball source